.github/workflows/bump_version_release.yaml

```
@@ -0,0 +1,75 @@
+name: Bump version and release
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        type: choice
+        required: true
+        options:
+          - "RF24"
+          - "RF24Network"
+          - "RF24Mesh"
+          - "RF24Gateway"
+          - "RF24Ethernet"
+      major:
+        type: boolean
+        default: false
+        required: true
+      minor:
+        type: boolean
+        default: false
+        required: true
+      patch:
+        type: boolean
+        default: true
+        required: true
```

Since it isn't very likely that we'd need to bump more than 1 version component at a time, I think these inputs can be consolidated:
```suggestion
      bump-component:
        type: choice
        default: 'patch'
        required: true
        options:
          - 'major'
          - 'minor'
          - 'patch'
```